src/app/components/media/MediaActionsBar.js

```diff
@@ -103,7 +103,17 @@ class MediaActionsBarComponent extends Component {
   }
 
   currentProject() {
-    return this.props.media.project;
+    let project = null;
+    let currentProjectId = window.location.pathname.match(/project\/([0-9]+)/);
+    if (currentProjectId) {
+      currentProjectId = parseInt(currentProjectId[1], 10);
+      project = this.props.media.projects.edges.filter(p =>
+        parseInt(p.node.dbid, 10) === currentProjectId);
+      if (project.length) {
+        project = project[0].node;
+      }
+    }
+    return project;
   }
```

Comment:
This is basically the same function as above... can we extract it as a function to the helpers file? I guess we'll need to use it in other places too.

Reply:
yes I used it in other places